meta-facebook/yv35-gl/src/platform/plat_def.h

```diff
@@ -17,6 +17,7 @@
 #ifndef PLAT_DEF_H
 #define PLAT_DEF_H
 
+#define ENABLE_ASD
```

Comment:
I don't see this #define getting used?

Reply:
Hi @jagpalgill 
The #define is used in oem_1s_handler.c.
https://github.com/facebook/OpenBIC/blob/main/common/service/ipmi/oem_1s_handler.c#L2378